doc/authorization-config.md

```diff
@@ -0,0 +1,385 @@
+not, at present, specifically given to `allow` or `deny` as a result.  See
+[TK-268](https://tickets.puppetlabs.com/browse/TK-268).
+
+One of the following forms may be used as the value for an `allow` entry:
+
+* An exact name.  For example: `www.domain.org`.  In this case, only a 
+ request whose "name" were `www.domain.org` would be considered a match for 
+ the entry.
+  
+* A glob of names, with an asterisk, `*`, in place of the leftmost segment. 
+ For example: `*.domain.org`.  In this case, either `www.domain.org` or 
+ `test.domain.org` would be considered a match for the entry.
+  
+* A regular expression, with surrounding solidus, `/`, characters.  For 
+ example: `/domain/`.  In this case, `www.domain.org`, `test.domain.org`, or 
+ `www.domain.com` would be considered a match for the entry.
+  
+* A backreference to a capture group, only applicable when used with a rule 
+ whose `type` is `regex`.  For example, if the `path` for the rule were 
+`"^/the/path/([^/]+)$"`, a backreference to the first capture group in the 
+ regular expression could be made by using a value like `"$1.domain.org"`.  
+ In this case, if the authenticated user's "name" were `www.domain.org` and 
+ the request URL were `"http://my-host:8080/the/path/www"`, the authenticated 
+ "name" would be considered a match for the entry.  An authenticated "name" 
+ of `xyz.domain.org` and a request URL of
+ `"http://my-host:8080/the/path/xyz"`, however, would not be considered a 
+ match for the entry.
+
+#### `deny`
+
+One of `allow`, `deny`, or `allow-unauthenticated` is required to be present
+for a rule.  If `allow-unauthenticated` is set to `true` for a rule, `deny` 
+may not be used.
+
+The value for a `deny` setting can be represented either as a single string 
+value ...
+
+~~~~hocon
+deny: node1
+~~~~
+
+... or as an array of values ...
+
+~~~~hocon
+deny: [ node1, node2, node3 ]
+~~~~
+
+If a request matches the criteria in the [`match-request`] (#match-request) 
+section of the rule and the authenticated "name" of the request matches one 
+of the `deny` entries, the request is "denied" - in which case the handler returns
+an HTTP 403/Forbidden response.  See the
+[`allow-header-cert-info`] (#allow-header-cert-info) setting for information on
+how the authenticated "name" is derived for a request.
+
+Note that if both `allow` and `deny` settings are included in the rule and 
+the authenticated "name" matches an entry in both settings, precedence is 
+not, at present, specifically given to `allow` or `deny` as a result.  See
+[TK-268](https://tickets.puppetlabs.com/browse/TK-268).
+
+The supported forms for a `deny` entry are the same as those for an `allow` 
+entry.  See documentation for the [`allow`] (#allow) setting for more 
+information.
+
+#### `allow-unauthenticated`
+
+One of `allow`, `deny`, or `allow-unauthenticated` is required to be present
+for a rule.  If `allow-unauthenticated` is set to `true` for a rule, neither
```

Comment:
The value doesn't matter, just whether `allow-unauthenticated` is present or not